plugin/src/libs/env.ts

```diff
@@ -0,0 +1,153 @@
+// Thanks Datadog! https://github.com/DataDog/serverless-plugin-datadog/blob/9c9e8fc78b51f752df4e1958b938c82e561a20a3/src/env.ts
```

Comment:
future: would this be something that people writing other plugins like this one would find helpful to live in a micro lib?

Reply:
I reckon a library that tries to standardise writing plugins for serverless and provides helper functions like this might be useful.

It could provide the consistent well documented plugin API that serverless doesn't (with types etc.) Probably a lot of work to develop and maintain though. A micro lib with helper functions would probably be more reasonable.